website/src/components/discourse/discourse.cy.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { DiscourseFeed } from './index'
+
+const mockResponseData = [
+    {
+        "id": 4911,
+        "title": "Processing source tables with differing arrival times throughout day",
+        "has_accepted_answer": true,
+        "author": "MatthewMunn",
+        "like_count": 1,
+        "posts_count": 2
+    },
+]
+
+describe('Test Discourse component', () => {
+    it('Should render same text passed into title prop', () => {
+        cy.mount(<DiscourseFeed title='Open topics' />)
+        cy.contains('Open topics').should('be.visible')
+    })
+
+    it('Should display loading icon on inital load', () => {
+        cy.intercept('POST', 'http://localhost:8888/.netlify/functions/get-discourse-topics', {
+            delayMs: 10,
+        }
```

Comment:
Question on the url here, does `cy.intercept` actually hit the endpoint here? Or is it just an example request? Wondering if it needs to be adjusted for when live. If it's just an example then i don't see a need to update it!

Reply:
oh yea, that is a good call out. I don't know if living the url as is will be a breaking change when the test are in production. I would need to do some research on that.  However,  I thing I read somewhere I can use regex to make that more resilient to different url environments. Cypress intercept does like urlPatternMatch function.